feat: Add AltSource

auggeeo opened the pull request:
This pull request adds an AltSource to make it easier for people using AltStore or SideStore!

auggeeo commented:
The newly added things are complete

RoootTheFox commented on .github/workflows/static.yml:
```diff
@@ -7,16 +7,16 @@ on:
 
 jobs:
   deploy:
-    runs-on: ubuntu-latest
```
why is this removed

RoootTheFox commented on .github/workflows/static.yml:
```diff
@@ -7,16 +7,16 @@ on:
       contents: write
     concurrency:
       group: ${{ github.workflow }}-${{ github.ref }}
+    runs-on: ubuntu-latest
```
oh nvm, lmao

RoootTheFox commented on .github/workflows/static.yml:
```diff
@@ -65,7 +65,17 @@ jobs:
+          gh_token: ${{ secrets.GITHUB_TOKEN }}
+          REPO_OWNER: ${{ github.repository_owner }}
+
+      - name: Submit to branch for pages to collect
```
imo this should be called "deploy" (as that is what it does), makes it more understandable

auggeeo replied:
ight!

RoootTheFox commented on altsource/baseapp.json:
```diff
@@ -0,0 +1,45 @@
+      "localizedDescription": "Geode, has come to iOS!",
+      "iconURL": "https://raw.githubusercontent.com/geode-sdk/ios-launcher/refs/heads/main/Resources/1024.png",
+      "tintColor": "#272727",
+      "category": "games",
+      "screenshots": [
+        {
+          "imageURL": "https://ios-repo.geode-sdk.org/img/LauncherShotiPhone-1.png",
+          "width": 1242,
+          "height": 2208
+        },
+        {
+          "imageURL": "https://ios-repo.geode-sdk.org/img/LauncherShotiPhone-2.png",
+          "width": 2208,
+          "height": 1242
+        },
+        {
+          "imageURL": "https://ios-repo.geode-sdk.org/img/LauncherShotiPhone-3.png",
+          "width": 2208,
+          "height": 1242
+        }
+      ],
+      "versions": [
+        {
+          "version": "0.5.1",
+          "minOSVersion": "14.0",
+          "localizedDescription": "## Changelog\r\n- Added Trollstore warning to tell you to enable \"URL Scheme\" in settings\r\n- Removed Nightly being enabled by default\r\n- Added landscape rotation support for phones\r\n- Fixed bug where cancelling the update early for Geode will assume Geode was updated for nightly releases\r\n- Fixed Trollstore version of the launcher crashing when going t ...
+          "date": "2025-05-03T16:14:15Z",
+          "downloadURL": "https://github.com/geode-sdk/ios-launcher/releases/download/v0.5.1/com.geode.launcher_0.5.1.ipa",
+          "size": 3248860
+        }
+      ],
+      "appPermissions": {
+        "entitlements": [
+          "filled automatically"
+        ],
+        "privacy": {
+          "FilledAutomatically": "This is filled in automatically."
+        }
+      }
+    }
```
i'd prefer if this was autogenerated (like the others) but it'll suffice for now

RoootTheFox commented on public/index.html:
```diff
@@ -19,41 +19,103 @@
-            <a class="btn btn-primary" href="apt-repo://https://ios-repo.geode-sdk.org"
-              >Add to Saily</a
-            >
+              href="sidestore://source?URL=https://ios-repo.geode-sdk.org/altsource/pre.json"><!-- DON'T FORGET TO CHANGE! -->
```
DONT FORGET TO CHANGE!

auggeeo replied:
oops lmao

RoootTheFox reviewed (commented):
bwa

auggeeo committed:
feat: Add AltSource

This commit adds an AltSource for AltStore (and forks of it, ex: SideStore) to be used as a easy updating and installation method

RoootTheFox commented:
fuck it we ball, lgtm, goob
